ManagedApp/mainTemplate.json

```diff
@@ -273,7 +273,7 @@
               "tableName": "BasicDataSet_v0.Message_v0",
               "predicate": {
                 "type": "Expression",
-                "value": "SentDateTime > @{dataset().windowStart} AND SentDateTime  <= @{dataset().windowEnd}"
```

Comment:
We used to use SentDateTime for messages earlier. What's the reason to use ReceivedDatetime elsewhere? We should keep it consistent to what we are prescribing as a default. If Received makes more sense - let's update it here.

Reply:
Same as above